feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature-1 branch
```

Comment:
This is good change and the application is working as per requirement.
Great 
![P](https://user-images.githubusercontent.com/76045075/103423631-a98e8280-4bcd-11eb-83b9-b23c7fe1f080.jpg)

Reply:
Thanks Pramod for your review.